src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/MarkerVisual/QRCodeMarkerVisualDetectorSpatialLocalizer.cs

```diff
@@ -16,8 +16,8 @@ protected override bool IsSupported
         {
             get
             {
-#if QRCODESTRACKER_BINARY_AVAILABLE && UNITY_WSA
-                return true;
+#if QRCODESTRACKER_BINARY_AVAILABLE && UNITY_WSA && !UNITY_EDITOR
+                return (Windows.ApplicationModel.Package.Current.Id.Architecture != Windows.System.ProcessorArchitecture.X86); // HoloLens 1 is not supported.
```

Comment:
>// HoloLens 1 is not supported. [](start = 127, length = 31)

Do you think this should check specifically for ARM instead of for !x86?

Reply:
hm, technically this may one day be available for x64 desktop scenarios. I think for now this will suffice.

Reply:
Smart!

---
In reply to: [305533927](https://github.com/microsoft/MixedReality-SpectatorView/pull/132#discussion_r305533927) [](ancestors = 305533927)